source/install/cloud-changelog.md

```diff
@@ -4,12 +4,70 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
-- [Release 2022-03-16](#release-2022-03-16)
+
+## Release 2022-06-15
+
+### Highlights
+
+#### Free Forever Mattermost Cloud Plan
+ - Mattermost Cloud now supports a free forever plan, Cloud Starter, for unlimited users for unlimited time.
+ - Access unlimited channels, playbooks, and boards across the Mattermost platform, with unlimited voice calls and screen sharing in Direct Messages.
```

Comment:
This sentence may read to some as if calls in v7.0 are only available via DMs, though calls is a Channels integration available across all public and private channels (I believe).

Reply:
I guess it depends on what we are referring to. 

- Cloud free (starter) plan has Calls limited to DMs only (essentially forcing 1-1s). 
- Cloud professional/enterprise have a temporary limit of 8 participants per call just for the time being (initial beta phase) but are not limited to any specific channel type.
- Self-hosted (any version/plan) has no limits.

Reply:
@jasonblais can confirm if we're just referring to Starter.

Reply:
This note is specific to Starter. There is a separate highlight section in the changelog for `Calls (Beta)` below.